lessons/frameworks.ipynb

```diff
@@ -0,0 +1,1261 @@
+    "\n",
+    "> \"68% of datasets would give a 68% frequentist confidence interval that contains the true parameter value\"\n",
+    "\n",
+    "> \"The probability of the true parameter value lying within the 68% Bayesian credible region is 68%\"\n",
+    "\n",
+    "* The difference in wording comes from the two different definitions of probability"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Uncertainties in the estimators\n",
+    "\n",
+    "* The covariance matrix of a Gaussian approximation to the likelihood defines a 1-sigma, 2D, elliptical, _frequentist confidence region_\n",
+    "\n",
+    "* Since this came from transforming the sampling distribution, which is a PDF over datasets, the confidence interval enables conclusions in terms of fractions of an ensemble of (hypothetical) datasets\n",
+    "\n",
+    "* The 68% confidence interval is the region that we expect to contain the true parameter value 68% of the time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism and Bayesianism\n",
+    "\n",
+    "* In frequentism, the data are considered to be random variables (in large sets of hypothetical trials described with probability distributions) while parameters are considered fixed (and to be estimated)\n",
+    "\n",
+    "* In Bayesianism, the data are considered to be fixed (as constants, in datafiles) while parameters are considered random variables (to be inferred, with uncertainty described by probability distributions)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism and Bayesianism\n",
+    "\n",
+    "Given an assumed model:\n",
+    "\n",
+    "* Frequentists seek to _transform_ the frequency distribution of the data into a frequency distribution of their estimators, and hence quantify their uncertainty in terms of _what they expect would happen if the observation were to be repeated_  \n",
+    "  \n",
+    "* Bayesians seek to _update their beliefs_ about their model parameters, and hence quantify their uncertainty in terms of _what might have been had the observation been different_, and _what they knew before the data were taken_\n",
+    "\n",
+    "#### Q: Which approach is better matched to science?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Frequentism vs. Bayesianism\n",
+    "\n",
+    "\n",
+    "* \"You have to assume a prior\" cf. \"You get to assume a prior\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* \"Your calculations are computationally expensive\""
```

Comment:
Just have a think about exactly what you want to say here (and what you want the notebook to say in isolation). E.g., the 2nd bullet isn't necessarily relevant because calculations can be expensive in both cases (doing profile likelihoods is usually worse than MCMC). But maybe that was the point.

Reply:
Yes - I was trying to make statements that either "side" could make - or otherwise draw a contrast for discussion.